app.ts

```diff
@@ -0,0 +1,272 @@
+  return noOfRows;
+};
+
+//// Functions To Create/Clear The HTML
+
+// Heading Row
+
+const createHeadingRow = (headingData: string) => {
+  const heading: any = document.querySelector("#heading");
+
+  let headings = `<div class="headings" id="headings">${headingData}</div>`;
+  heading.innerHTML += headings;
+};
+
+// Table Content
+
+const createTableContent = (contentData: string) => {
+  const content: any = document.querySelector("#content");
+
+  let table = `<div id="row-${contentData[0]}" class="rows"></div>`;
+  content.innerHTML += table;
+
+  let rows: any = document.querySelector("#row-" + contentData[0] + ".rows");
+  for (let x = 0; x < contentData.length; x++) {
+    let rowCols = `<div class="row_cols">${contentData[x]}</div>`;
+    rows.innerHTML += rowCols;
+  }
+};
+
+// Clear Table Content
+
+const clearTable = () => {
+  const content: any = document.querySelector("#content");
+  const clear = "";
+
+  content.innerHTML = clear;
+};
+
+//// Fetch Requests
+
+// Heading Row (Getting the columns data)
+
+const getHeadings = () => {
+  fetch("http://localhost:2050/columns", {
+    method: "GET",
+    headers: { "Content-Type": "application/json" },
+  })
+    .then((res) => res.text())
+    .then((data) => {
+      let headingData = JSON.parse(data);
+      for (let i = 0; i < headingData.length; i++) {
+        createHeadingRow(headingData[i]);
+      }
+      getParameters((fromParameter = 0));
+    })
+    .catch((error) => {
+      console.log("Call Hector", error);
+    });
+};
+
+// Table Content (Getting the table's data)
+
+const getTable = () => {
+  let toParameter = getParameters(fromParameter);
+
+  fetch(`http://localhost:2050/records?from=${fromParameter}&to=${toParameter}`, {
+    method: "GET",
+    headers: { "Content-Type": "application/json" },
+  })
+    .then((res) => res.text())
+    .then((data) => {
+      let contentData = JSON.parse(data);
+      for (let i = 0; i < contentData.length; i++) {
+        createTableContent(contentData[i]);
+      }
+    })
+    .catch((error) => {
+      console.log(error);
+    });
+};
+
+// Displays The Current Results Being Shown
+
+const stats = () => {
+  const pageStats: any = document.querySelector("#pageStats");
+  let toParameter = getParameters(fromParameter);
+
+  let currentStats = `Showing results from ${fromParameter} to ${toParameter} out of 1 000 000 results.`;
+  pageStats.innerHTML = currentStats;
+};
+
+//// Debounce
+
+const debounce = (fn: any, delay: number) => {
+  let timer: number;
+  return function () {
+    clearTimeout(timer);
+    timer = setTimeout(() => {
+      fn();
+    }, delay);
+  };
+};
+
+//// Sizing And Resizing
+
+let resizing = () => {
+  let end = fromParameter + getNoOfRows();
+  let toParameter: number;
+
+  if (end > 999999) {
+    toParameter = 999999;
+    fromParameter = toParameter - getNoOfRows();
+  } else {
+    toParameter = fromParameter + getNoOfRows();
+  }
+  clearTable();
+  getTable();
+  stats();
```

Comment:
I can definitely see these 3 functions actually being in one as it always goes together logically. They are always repeated in that order and are not too complex to be split up. It can be all together in getTable()?